tools/dependency/release_dates.py

```diff
@@ -14,17 +14,31 @@
 import os
 import sys
 import argparse
+import string
 
 import github
 
 import exports
 import utils
-
 from colorama import Fore, Style
 from packaging import version
 
 # Tag issues created with these labels.
 LABELS = ['dependencies', 'area/build', 'no stalebot']
+GITHUB_REPO_LOCATION = "ME-ON1/envoy"
```

Comment:
also just noticed that this is still pointing to your repo

Reply:
thanks for pointing it out!